lib/plurimath/math/core.rb

```diff
@@ -213,7 +213,12 @@ def linebreak?
       end
 
       def cloned_objects
-        object = self.class.new rescue self.class.new(nil)
+        object = begin
```

Comment:
Using rescue triggers a lot of overhead internally. Why not just do a const check and instantiate?